README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
 
-### Compensation
+- Informally, we say you have **equity** in a company when you have some kind of ownership or likely future ownership. In practice, this can mean, stock, stock options, and RSUs, which we’ll divide and conquer later.
+	- 👆 The word “equity” has [several technical meanings](https://www.investopedia.com/terms/e/equity.asp) in accounting and other financial contexts, but when we talk about equity compensation, it is meant more informally.
 
-- Your **compensation** is everything you get for working for a company.
-- When you negotiate compensation with a company, the elements to think about are cash (salary and bonus), benefits (health insurance, retirement, perks), and equity (what we discuss here).
-- [**Equity compensation**](http://www.investopedia.com/terms/e/equity-compensation.asp) refers to owning stock or having the right to buy stock in a company. In general, this Guide is focused on equity compensation in corporations, not limited liability companies. The reasons for this are:
-    - (i) Corporations are the most common form of startup company in the U.S. (LLCs are rarely used as the choice of entity for technology startups.)
-    - (ii) Equity compensation for limited liability companies is dramatically different from equity compensation in corporations.
-- Equity compensation is commonly used for founders, executives, employees, contractors, advisors, directors, and others.
-- The purpose of equity compensation is twofold:
-    - To attract the best talent
-    - To align individuals' incentives with the interests of the company.
-- Equity compensation generally consists of stock, stock options, or restricted stock units (RSUs) in the company. We’ll define these concepts next.
+- **Why give equity compensation?** The purpose of equity compensation is twofold:
+    - To attract the best talent: When a company already has or can be predicted to have significant financial success, the most talented people are incentivized to work for the company by the prospect of their equity turning into large amounts of cash in the future.
+    - To align individuals’ incentives with the interests of the company: Even companies that can afford to pay lots of cash may prefer to give employees equity, so that employees work to increase the *future* value of the company. At its best, this philosophy fosters an environment of teamwork and a “rising tides lift all boats” mentality. As we’ll talk more about during our chapter on [negotiations](negotiations-and-offers), the amount of equity you’re offered usually reflects both your past or current contribution to the company and your commitment to the company in the future.
```

Comment:
Internal link for "negotiations" needs a #

Reply:
✔️